src/ElmFive.elm

```diff
@@ -181,27 +181,23 @@ translations =
 
 translate : String -> String
 translate language =
-    case Dict.get language translations of
-        Nothing ->
-            "five"
-
-        Just translation ->
-            translation
+    Dict.get language translations
+        |> Maybe.withDefault "five"
 
 
 isFive : Int -> Bool
-isFive value =
-    value == five
+isFive =
+    (==) five
 
 
 filter : List Int -> List Int
-filter values =
-    List.filter (\value -> isFive value) values
+filter =
+    List.filter isFive
```

Comment:
This is one of those cases where I would definitely do it point-free, since the signature, coupled with a potentially better name (filterFives, or something) could make the `values` label unnecessary